test/integration/remove/test_remove.py

```diff
@@ -0,0 +1,86 @@
+import re
+import subprocess
+import os
+
+def test_remove_tt(tt_cmd, tmpdir):
+
+    configPath = os.path.join(tmpdir, "tarantool.yaml")
+     # Create test config with restart_on_failure true.
+    with open(configPath, 'w') as f:
+        f.write('tt:\n  app:\n    bin_dir:\n    inc_dir:\n')
+
+    # Install latest tarantool.
+    start_cmd = [tt_cmd, "install", "tt=master", "--cfg", configPath]
+    instance_process = subprocess.Popen(
+        start_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process was terminated correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+    installed_cmd = [tmpdir + "/bin/tt", "run", "-v"]
+    installed_programm_process = subprocess.Popen(
```

Comment:
Pls, use word "program" with one "m" everywhere. "Programm" is incorrect in English.

Reply:
Changed